test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12084,11 +12109,11 @@ def decay_fairshare_tree(self):
         if self.has_diag:
             return True
 
-        self.stop()
         pbsfs = os.path.join(self.pbs_conf['PBS_EXEC'], 'sbin', 'pbsfs')
+        if self.sc_name is not 'default':
+            cmd += ['-I', self.sc_name]
```

Comment:
command is loaded to pbsfs here , So -I sched_name should be added to pbsfs.

Reply:
oops, copy paste error.  Thanks for finding this.